internal/backuprestore/backup.go

```diff
@@ -565,3 +565,26 @@ func (h *BRHandler) ensureBackupsDeleted(ctx context.Context, backups []velerov1
 	}
 	return true, nil
 }
+
+// CleanupDeleteBackupRequests deletes all DeleteBackupRequest for this cluster from object storage
+// returns: true if all DeleteBackupRequest have been deleted, error
+func (h *BRHandler) CleanupDeleteBackupRequests(ctx context.Context) (bool, error) {
```

Comment:
I think only one return variable err is needed.
```suggestion
func (h *BRHandler) CleanupDeleteBackupRequests(ctx context.Context) error {
```